pages/forgotPassword/index.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components'
+import SpaceLogo from "components/SpaceLogo";
+import ForgotPasswordForm from 'components/auth/ForgotPasswordForm';
+
+const Main = styled.main`
+width: 100%;
+height: 100%;
+display: flex;
+min-height: 100vh;
+align-items: center;
+justify-content: center;
+background-size: cover;
+background-image: url("auth-background.jpg");
+`
+
+const ForgotPassword: React.FC = () => {
```

Comment:
Noticed an issue here, if you refresh page or go directly to /forgotPassword, you get stuck on modal, you can't use "X" to go-to login, this should act the same way in all situations.

I think AuthWrapper should be in this page too.